src/root/views/region.js

```diff
@@ -568,7 +568,7 @@ const selector = (state, ownProps) => ({
   appealsListStats: state.overallStats.appealsListStats,
   countriesByRegion: countriesByRegionSelector(state),
   regions: regionsByIdSelector(state),
-  thisRegion: regionByIdOrNameSelector(state, ownProps.match.params.id),
+  thisRegion: regionByIdOrNameSelector(state, ownProps.match.params.id.split('#')[0]),
```

Comment:
@frozenhelium should I remove this cutdown also?

Reply:
I did it in my latest push :D